linera-views/src/key_value_store_view.rs

```diff
@@ -67,6 +67,12 @@ impl SizeData {
         self.key + self.value
     }
 
+    /// Sums both terms
+    pub fn sum_i32(&mut self) -> Result<i32, ViewError> {
+        let sum = self.key + self.value;
```

Comment:
And, just for completeness, we could use `checked_add` here, too.

Reply:
ok, done.